test/e2e/lib/driver-manager.js

```diff
@@ -133,6 +134,8 @@ export async function startBrowser( { useCustomUA = true, resizeBrowserWindow =
 	} else {
 		switch ( browser.toLowerCase() ) {
 			case 'chrome':
+				const chromeVersion = await fs.readFileSync( './.chromedriver_version', 'utf8' ).trim();
```

Comment:
Should these 2 be defined inside the `if ( useCustomUA ) {` check? Because if `useCustomUA` is false, we'll still read from the chromedriver version file, but not use these variables at all?

Reply:
yes, that makes sense. updated.